### ISSUE TITLE
Help the GPflow community flourish

### ISSUE THREAD

**Issue by st--** [GPflow/GPflow#1216]
We would like to help the GPflow community be the best it can be. As part of this we want to
[ ] more publicly advertise the gpflow slack workspace (add to README.md / contributing.md [which may also want a check/update as it hasn't been touched in ages])
[ ] set up github and stackoverflow integration in the gpflow slack workspace to post new issues and questions, so that we more easily find out and can respond faster
[ ] get more collaborators involved on the GPflow code!

**Comment by markvdw** [comment]
I agree! The centre of gravity of GPflow development has largely become Prowler, but I do still want to be involved even now I'm not around anymore. The larger reliance on GitHub for tracking issues has been good, but the GPflow slack would be even better.

**Comment by st--** [comment]
Re Slack workspace: can we make it so that one doesn't need an invite to join ? Otherwise we would have to provide a way for people to leave their e-mail addresses, AND monitor it...

**Comment by st--** [comment]
Slack workspace now has a public invite link in the README.md, this got added in #1225. :)

**Comment by st--** [comment]
Stackoverflow integration has been added to gpflow.slack.com as well
